ui/src/components/fields/accordion.js

```diff
@@ -34,13 +34,18 @@ class Accordion extends Component {
 			'dashicons-arrow-right-alt2': true,
 		});
 
+		const expanded = this.state.active ? 'true' : 'false';
+
+		const escapedLabel = this.props.label.replace(/"/g, '\\\\\"');
+		const buttonLabel = `${this.state.active ? 'Collapse' : 'Expand'} ${escapedLabel}`;
```

Comment:
Probably want to make these strings translatable?

Reply:
I agree but, per my previous comment, I don't know how y'alls setup handles translation.

Reply:
You can add your localized strings [here](https://github.com/moderntribe/panel-builder/blob/12c2ba3c39bc0e2f4d52a246813d3d63f203a7fd/ModularContent/MetaBox.php#L121) and then use them in your JS [like](https://github.com/moderntribe/panel-builder/blob/589e8a3c49bcc9d8bcd1a06044d8a8cc95937e89/ui/src/components/shared/panel-set-preview.js#L4) [so](https://github.com/moderntribe/panel-builder/blob/589e8a3c49bcc9d8bcd1a06044d8a8cc95937e89/ui/src/components/shared/panel-set-preview.js#L51).